### PR TITLE
Create mentalHealthBot.py

### PR DESCRIPTION
**Pull request by joeyouss**
Created mental health monitoring bot

### CODE REVIEW

**Review by georgeamccarthy — COMMENTED** [review]
Looks good.

File: mentalHealthBot.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
+# # bot.py
+import json
+import os
+import discord
+import requests
+from discord.ext import commands
+import random
+
+# enter token for the server
+TOKEN = ''
```
Comment (georgeamccarthy) [inline mentalHealthBot.py]: Use .env file to hide key.
```suggestion
from dotenv import load_dotenv
load_dotenv()<br class="Apple-interchange-newline">
TOKEN = os.getenv('DISCORD_TOKEN')
```

File: mentalHealthBot.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+# enter application id
+GUILD = ''
```
Comment (georgeamccarthy) [inline mentalHealthBot.py]: rm, guild not used.
```suggestion

```

**Review by georgeamccarthy — APPROVED** [review]

File: mentalHealthBot.py (hunk 1 above)
Comment (SaurabhAgarwala) [reply mentalHealthBot.py]:
```suggestion
from dotenv import load_dotenv
load_dotenv()
TOKEN = os.getenv('DISCORD_TOKEN')
```

Comment (SaurabhAgarwala) [reply mentalHealthBot.py]: I felt there was a small typo, just corrected it

File: mentalHealthBot.py [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+        r1 = "And just to brighten your day, here is a joke for you"
+        await message.channel.send(r1)
+        r2 = joke()
+        await message.channel.send(r2)
+
+    if 'quote' in message.content.lower():
+        response = get_qoute()
+        await message.channel.send(response)
+
+client.run(TOKEN)
```
Comment (SaurabhAgarwala) [inline mentalHealthBot.py]: This is great, but I think the previous code is missing from here. So it needs to be updated, else there will be conflicts. Also, in the existing code `Bot` has been used, but here `Client` is being used. So both needs to be made uniform, for all functionalities to work.

**Review by SaurabhAgarwala — COMMENTED** [review]
Added a few comments which needs to be solved before merging.

**Review by SaurabhAgarwala — APPROVED** [review]

**Pull request merged**